aspnetcore/fundamentals/w3c-logger/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: W3C Logger in .NET Core and ASP.NET Core
+author: wtgodbe
+description: Learn how to create server logs in the W3C standard format.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: wigodbe
+ms.custom: mvc
+ms.date: 09/29/2021
+no-loc: [Home, Privacy, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: fundamentals/w3c-logger/index
+---
+
+# W3C Logger in ASP.NET Core
```

Comment:
Nit: Consistency of name (I think it should be W3CLogger everywhere, with that casing and no spaces)